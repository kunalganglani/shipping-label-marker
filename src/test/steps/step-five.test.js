import React from 'react';
import renderer from 'react-test-renderer';
import Confirm from '../../features/shipping-label-maker/steps/step-five';
test('Confirm renders correctly', () => {
  const wizardContext = {
    from: {
      name: '',
      street: '',
      city: '',
      state: '',
      zip: '',
    },
    to: {
      name: '',
      street: '',
      city: '',
      state: '',
      zip: '',
    },
    weight: {
      weightValue: '',
      weightUnit: 'Pounds',
    },
    shippingOption: '1',
  };
  const handleAction = jest.fn();
  const tree = renderer
    .create(<Confirm wizardContext={wizardContext} onAction={handleAction} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
