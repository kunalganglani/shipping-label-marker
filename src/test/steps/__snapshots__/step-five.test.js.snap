// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confirm renders correctly 1`] = `
<div
  className="css-1a6kald"
>
  <h3>
    Shipping Information
  </h3>
  <h4>
    Shipping Option: GROUND
  </h4>
  <h4>
    Weight:  Pounds
  </h4>
  <h4>
    Cost: $ 0.00
  </h4>
  <h3>
    Sender's Information
  </h3>
  <h4>
    Name: 
  </h4>
  <h4>
    Street: 
  </h4>
  <h4>
    City: 
  </h4>
  <h4>
    State: 
  </h4>
  <h4>
    Zip: 
  </h4>
  <h3>
    Receiver's Information
  </h3>
  <h4>
    Name: 
  </h4>
  <h4>
    Street: 
  </h4>
  <h4>
    City: 
  </h4>
  <h4>
    State: 
  </h4>
  <h4>
    Zip: 
  </h4>
</div>
`;
